client/src/index.js

```diff
@@ -1,18 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import Form from './components/Form.jsx';
+import axios from 'axios';
 class App extends React.Component {
   constructor() {
     super();
     
     this.state = {
       title: ''
     };
+    this.asyncGet = this.asyncGet.bind(this);
   }
 
+  asyncGet(callback) {
+    axios.get('/api/buytest')
+      .then(function (response) {
+        // console.log('THA RESPONSE', response);
```

Comment:
You can remove the console log here and on line 22

Reply:
Thanks, I'll make the changes